src/Routing/Router.php

```diff
@@ -594,6 +594,8 @@ public static function url($url = null, $full = false)
                 unset($url['_ssl']);
             }
 
+            $url = static::_applyUrlFilters($url);
```

Comment:
One can do the same thing that's done in this method to get context if needed, use `Router::getRequest()`